core/src/main/scala/com/pingcap/tispark/write/TiDBOptions.scala

```diff
@@ -282,6 +286,7 @@ object TiDBOptions {
   val TIDB_ENABLE_UPDATE_TABLE_STATISTICS: String = newOption("enableUpdateTableStatistics")
   val TIDB_DEDUPLICATE: String = newOption("deduplicate")
   val TIDB_ROWID: String = newOption("tidbRowId")
+  val TiDB_ROW_FORMAT_VERSION: String = newOption("rowFormatVersion")
```

Comment:
Why a new mechanism was introduced instead of following the past practice?https://github.com/pingcap/tispark/blob/cc9c121dab6dff68fbb7ca1cae4c6b662845c7b8/core/src/main/scala/com/pingcap/tispark/write/TiBatchWriteTable.scala#L55-L56

Reply:
we decide to remove jdbc client and only use it to update statistics. This option was gotten by jdbc client, so we are not able to get it now. Instead, we put it in TiDBOption and users can set it by `spark.conf.set`.